Makefile

```diff
@@ -41,10 +41,7 @@ install-smoketest:
 	$(MAKE) -C api release
 	$(MAKE) -C runner release
 
-apply-smoketest:
-	$(MAKE) -C setup apply-smoketest
-
 destroy-smoketest:
```

Comment:
we should remove `destroy-smoketest` as well

Reply:
✅